cookbooks/redis/README.md

```diff
@@ -0,0 +1,73 @@
+## Optional Cookbook for Engine Yard Cloud
+
+# Redis
+
+[Redis][1] Redis is an open source, advanced key-value store. It is often referred to as a data structure server since keys can contain [strings][7], [hashes][6], [lists][5], [sets][4] and [sorted sets][3].  Learn More at the [introduction][7].
+
+## Overview
+
+This cookbook provides a method to host a dedicated redis instance.  This recipe should *NOT* be used on your Database instance as it is not designed for that instance.  Additionally it will not change nor modify your database instance in anyway what so ever.
+
+## Installation
+
+Since this is an optional recipe, it should be installed by simply including it via a `depends` in your `ey-custom/metadata.rb` file and an `include_recipe` in the appropriate hook file. For more details on optional recipes see the [redis example]. (Yes, how convenient that the example is for the exact recipe you wanted)
+
+Additionally, this recipe will only activate on insta
+
+Simply add a utility instance named `redis` and the recipe will use that instance for redis. If the utility instance you wish to use redis on isn't called `redis`, update redis/attributes/default.rb with the correct instance name:
+
+```ruby
+default['redis'] = {
+  'utility_name' => "my_custom_name", # default: redis
+  # ...
+}
+```
+
+Ensure you have the Dependencies installed in your local cookbooks repository ...
+Add the following to your main/recipes/default.rb
```

Comment:
The reference to `main/recipes/default.rb` is my fault, carried over from https://github.com/engineyard/ey-cookbooks-stable-v5-DELETEME/blob/master/cookbooks/redis/README.md. 

I'll put in a PR to fix this as soon as this PR is merged.